conjure-java-core/src/integrationInput/java/com/palantir/product/SafeLongExample.java

```diff
@@ -135,6 +135,13 @@ public Builder safeLongList(@Nonnull Iterable<SafeLong> safeLongList) {
             return this;
         }
 
+        public Builder addAllSafeLongList(@Nonnull long... safeLongList) {
+            checkNotBuilt();
+            ConjureCollections.addAllToSafeLongList(
+                    this.safeLongList, Preconditions.checkNotNull(safeLongList, "safeLongList cannot be null"));
+            return this;
+        }
```

Comment:
I thought we weren't adding public API taking `long` as `SafeLong` -- given that this doesn't quite match the other cases where we have an exact primitive type to operate upon, I think we should consider this case separately.

I'm all for making the non-public-api changes to optimize this case, but would prefer not to add overloads to generated public builders for it.